### PR TITLE
Added NFS Example

### PR DESCRIPTION
**Pull request by SrinivasChilveri**
Added nfs usage example 
Fixed from gcs to genectl in read me of examples

Fix Issue: https://github.com/kubegene/kubegene/issues/36

### CODE REVIEW

File: example/execution/README.md [hunk 1]
```diff
@@ -18,3 +18,37 @@ $ kubectl create -f exec-2.yaml
 $ kubectl create -f exec-3.yaml
 $ kubectl create -f iterate-exec.yaml
 ```
+
+The below example is with the nfs
+
+## Prerequisites
+
+ * k8s is deployed with minikube in a machine /laptop.
```
Comment (hzxuzhonghu) [inline example/execution/README.md]: This is not a must

File: example/execution/README.md [hunk 2]
```diff
@@ -18,3 +18,37 @@ $ kubectl create -f exec-2.yaml
...
+     just clone and build the project
+     $ cd $GOPATH/src/github.com/kubernetes-incubator
+     $ git clone https://github.com/kubernetes-incubator/external-storage.git
+     $ cd $GOPATH/src/github.com/kubernetes-incubator/external-storage/nfs
+     $ make
+     brought up the nfs-provisioner as shown below (supposed to create /export folder in that machine where all the data is stored)
+     $ sudo ./nfs-provisioner -provisioner=example.com/nfs
+     -kubeconfig=$HOME/.kube/config
+     -run-server=false
+     -use-ganesha=false
```
Comment (hzxuzhonghu) [inline example/execution/README.md]: I prefer abstract these in a shell script, just execute a script here.

File: example/group-job/README.md [hunk 3]
```diff
@@ -16,5 +16,5 @@ This is a simple example to demonstrate how to submit a group job using `genectl
 ## Command
 
 ```bash
-$ gcs sub job /tmp/subrepjob/repjob.sh  --tool nginx:latest --pvc subrepjob-pvc
+$ genectl sub job /tmp/subrepjob/repjob.sh  --tool nginx:latest --pvc subrepjob-pvc
```
Comment (hzxuzhonghu) [inline example/group-job/README.md]: good catch

File: example/execution/README.md (hunk 1 above)
Comment (hzxuzhonghu) [reply example/execution/README.md]: We just need a deployed k8s, no matter how it is deplyed and on what platform.

Comment (SrinivasChilveri) [reply example/execution/README.md]: Fixed, pls do check the fix.

File: example/execution/README.md (hunk 2 above)
Comment (SrinivasChilveri) [reply example/execution/README.md]: done

**Pull request merged**